packages/lab/src/carousel/Carousel.tsx

```diff
@@ -1,170 +1,47 @@
-import {
-  Button,
-  GridLayout,
-  RadioButton,
-  RadioButtonGroup,
-  makePrefixer,
-  useIcon,
-  useId,
-} from "@salt-ds/core";
-import { useComponentCssInjection } from "@salt-ds/styles";
-import { useWindow } from "@salt-ds/window";
-import { clsx } from "clsx";
-import {
-  type ChangeEventHandler,
-  Children,
-  type HTMLAttributes,
-  type ReactElement,
-  forwardRef,
-  useEffect,
-} from "react";
-import { DeckLayout } from "../deck-layout";
-import { useSlideSelection } from "../utils";
-import type { CarouselSlideProps } from "./CarouselSlide";
-
-import carouselCss from "./Carousel.css";
-
-const withBaseName = makePrefixer("saltCarousel");
+import { type HTMLAttributes, forwardRef } from "react";
+import { CarouselProvider } from "./CarouselContext";
 
 export interface CarouselProps extends HTMLAttributes<HTMLDivElement> {
   /**
    * The initial Index enables you to select the active slide in the carousel.
    * Optional, default 0.
    **/
-  initialIndex?: number;
-  /**
-   * The animation when the slides are shown.
-   * Optional. Defaults to `slide`
-   **/
-  animation?: "slide" | "fade";
+  activeSlideIndex?: number;
   /**
    * If this props is passed it will set the aria-label with value to the carousel container.
    * Optional. Defaults to undefined
    */
   carouselDescription?: string;
   /**
-   * Collection of slides to render
-   * Component must implement CarouselSlideProps. Mandatory.
-   */
-  children: Array<ReactElement<CarouselSlideProps>>;
-  /**
-   * This prop will enable compact / reduced width mode.
-   * The navigation buttons would be part of indicators
-   * Optional. Defaults to false
-   **/
-  compact?: boolean;
-  /**
-   * It sets the id for the Carousel Container.
-   * String. Optional
+   * A boolean. When `true`, the slider will receive a full border.
    */
-  id?: string;
+  bordered?: boolean;
```

Comment:
Styles might need to be checked, currently this shifts the alignment

Reply:
that's intentional, but the prop name might not give it very clearly... I will check this with @ivan-calderon but there is a Hero (transparent) version and a...norther(bordered) one hahaha will check the naming